Sources/Amplitude/Utilities/Session.swift

```diff
@@ -1,87 +0,0 @@
-//
-//  Session.swift
-//
-//
-//  Created by Alyssa.Yu on 12/2/22.
-//
-
-import Foundation
```

Comment:
I feel separate the session-related functions makes the code cleaner.  Any reason about moving this into the Timeline.swift?